osbs/api.py

```diff
@@ -378,13 +379,15 @@ def _create_build_config_and_build(self, build_request):
             logger.debug('build config for %s already exists, updating...',
                          build_config_name)
 
+            # Temporarily removing lowpriority nodeselector for explicit build
+            # it will be set back, once we create build
+            old_low_prio_ns = existing_bc['spec'].get('nodeSelector', {}).pop('lowpriority', None)
```

Comment:
well even if it won't be defined, this will return None

and later we won't be patching it back because it was None